README.md

```diff
@@ -69,7 +69,7 @@ run `$ yarn run test -- <path>`.
 
 #### Running back-end tests
 
-To run all tests, run `$ stack test`, or equivalently `$ stack test :Tests`.
+To run all tests, run `$ stack test`. To run a specific test group or test case, run `stack test --ta '-p "<label for group/case>"'`.
```

Comment:
This is good, but the concept of a "label" might be a bit confusing. I would add a sentence like "For more information on the `-p` argument, see https://github.com/UnkindPartition/tasty?tab=readme-ov-file#patterns."